libvcs4j-mapping/src/main/java/de/unibremen/informatik/st/libvcs4j/mapping/Mapping.java

```diff
@@ -31,6 +37,39 @@
 	 */
 	public static class Result<T> {
 
+		/**
+		 * Stores the ordinal of the range ({@link RevisionRange#getOrdinal()})
+		 * that was passed to
+		 * {@link Mapping#map(Collection, Collection, RevisionRange)}.
+		 */
+		private int ordinal;
+
+		/**
+		 * Stores the computed mapping result from
+		 * {@link Mapping#map(Collection, Collection, RevisionRange)}.
+		 */
+		private Map<Mappable<T>, Mappable<T>> mapping = new IdentityHashMap<>();
+
+
+		/**
+		 * Stores all mappables, whose lifespans are ending. These are all the
```

Comment:
The name of this attribute is `startingLifespans`, but the doc says: `whose lifespans are ending`. Furthermore, the doc of `getStartingLifespans` says: `Returns the list of all mappables, whose lifespans are starting.`. I think `ending` should be `starting` right?